cassandra/cqltypes.py

```diff
@@ -250,6 +252,7 @@ def lookup_casstype(casstype):
     try:
         return parse_casstype_args(casstype)
     except (ValueError, AssertionError, IndexError) as e:
+        log.debug("Exception in parse_casstype_args: %s" % e)
         raise ValueError("Don't know how to parse type string %r: %s" % (casstype, e))
 
 
```

Comment:
What is the reason for adding this debug log?

Reply:
It was part of https://github.com/datastax/python-driver/pull/1161, removed.